scripts/generate-docs-admin.ts

```diff
@@ -0,0 +1,14 @@
+import {components, extensionPoints} from './typedoc/shopify-dev-renderer';
+
+const paths = {
+  inputRoot: './packages/argo-admin',
+  packages: {
+    JavaScript: './packages/argo-admin',
+    React: './packages/argo-admin-react',
+  },
+  outputRoot: '../shopify-dev/content/tools/argo-admin',
```

Comment:
Should we not put these docs in the same place as Checkout like `docs/admin-extensions`?

Reply:
probably yes, haven't finalized this yet. not concerned about this right now.